.github/workflows/build-test.yml

```diff
@@ -52,55 +43,21 @@ jobs:
           echo PR_COMMIT_MESSAGE=$(git log -1 --pretty=format:\"%s\") >> $GITHUB_ENV
         shell: bash
 
-      - name: Set up Python ${{ matrix.python }}
-        uses: actions/setup-python@v5
+      - uses: prefix-dev/setup-pixi@v0.8.1
```

Comment:
In a next iterations, you can add something like this actions: https://github.com/skrub-data/

It allows to run the `pixi update` via an Actions and open a pull-request